docs/OV_Runtime_UG/preprocessing_usecase_save.md

```diff
@@ -0,0 +1,84 @@
+# Use Case - Integrate and Save Preprocessing Steps Into IR {#openvino_docs_OV_Runtime_UG_Preprocess_Usecase_save}
+
+## Introduction
+
+In previous sections we've covered how to add [preprocessing steps](./preprocessing_details.md) and got the overview of [Layout](./layout_overview.md) API.
+
+For many applications it is also important to minimize model's read/load time, so performing integration of preprocessing steps every time on application startup after `ov::runtime::Core::read_model` may look not convenient. In such cases, after adding of Pre- and Post-processing steps it can be useful to store new execution model to Intermediate Representation (IR, .xml format).
+
+Most part of existing preprocessing steps can also be performed via command line options using Model Optimizer tool. Refer to [Model Optimizer - Optimize Preprocessing Computation](../MO_DG/prepare_model/Additional_Optimizations.md) for details os such command line options.
+
+## Code example - saving model with preprocessing to IR
+
+In case if you have some preprocessing steps which can't be integrated into execution graph using Model Optimizer command line options (e.g. `YUV->RGB` color space conversion, Resize, etc.) it is possible to write simple code which:
+ - Reads original model (IR, ONNX, Paddle)
+ - Adds preprocessing/postprocessing steps
+ - Saves resulting model as IR (.xml/.bin)
+
+Let's consider the example, there is an original `ONNX` model which takes one `float32` input with shape `{1, 3, 224, 224}` with `RGB` channels order, with mean/scale values applied. User's application can provide `BGR` image buffer with not fixed size. Additionally, we'll also imagine that our application provides input images as batches, each batch contains 2 images. Here is how model conversion code may look like in your model preparation script
+
+- Includes / Imports
+@sphinxdirective
+
+.. tab:: C++
+
+      .. doxygensnippet:: docs/snippets/ov_preprocessing.cpp
+         :language: cpp
+         :fragment: [ov:preprocess:save_headers]
+
+.. tab:: Python
+
+      .. doxygensnippet:: docs/snippets/ov_preprocessing.py
+         :language: python
+         :fragment: [ov:preprocess:save_headers]
+
+@endsphinxdirective
+
+- Preprocessing & Saving to IR code
+
+@sphinxdirective
```

Comment:
because of the empty line between `sphinxdirective` and line above - the `Preprocessing & Saving to IR code` and `Includes / Imports`  snippets are rendered differently

Reply:
Fixed